src/webhooks/discord/templates/creatorRegistered.ts

```diff
@@ -0,0 +1,40 @@
+import { Prisma } from '@prisma/client';
+import { MessagePayload } from 'discord.js';
+
+export const CREATOR_REGISTERED = (
+  creator: Prisma.CreatorCreateInput,
+  apiUrl: string,
+  payload: MessagePayload,
+): MessagePayload => {
+  payload.body = {
+    embeds: [
+      {
+        title: '🚀 New Creator Registration',
+        description: 'A new creator has registered. Here are the details:',
+        color: 0x4caf50,
+        fields: [
+          {
+            name: 'Name',
+            value: creator.name,
+            inline: false,
+          },
+          {
+            name: 'Email',
+            value: creator.email,
+            inline: false,
+          },
+          {
+            name: 'Registration Time',
+            value: creator.createdAt.toString(),
+            inline: false,
```

Comment:
Can we display this in a short format? Instead of the full ISO format we can just use `23.11.2024. 13:11`

You can use the `date-fns` library for date management that we have as a dependency already. It has human readable dates